src/bosh_azure_cpi/spec/unit/vm_manager/get_storage_account_spec.rb

```diff
@@ -4,9 +4,8 @@
 
 describe Bosh::AzureCloud::VMManager do
   describe '#get_storage_account_from_vm_properties' do
-    let(:props_factory) { Bosh::AzureCloud::PropsFactory.new(Bosh::AzureCloud::ConfigFactory.build(mock_cloud_options)) }
+    let(:props_factory) { Bosh::AzureCloud::PropsFactory.new(Bosh::AzureCloud::ConfigFactory.build(mock_cloud_options['properties'])) }
```

Comment:
Not sure why `['properties']` is added here, the `mock_cloud_options` method doesn't seem to have changed.

https://github.com/cloudfoundry/bosh-azure-cpi-release/pull/698/files#diff-1a1fbe20391f7dce08027421a87d3efb885328926612e3823ccf57a05259b7f0L60-L92

Reply:
It hasn't changed, but this Config that was being created was always wrong previously. It passes either way now, but for a moment I was having the CPI raise a `Director is too old and did not provide stemcell api version` if the stemcell API wasn't passed in the config body. (the stemcell API version started getting passed as part of CPI API v2)

That caused these tests to fail because [mock_cloud_options](https://github.com/cloudfoundry/bosh-azure-cpi-release/blob/928e882f361145a39143f776cfdfff17156b2865/src/bosh_azure_cpi/spec/spec_helper.rb#L63) isn't really the config body, but the `properties` inside of it are.

I later removed the `Director is too old and did not provide stemcell api version` error because it turns out there are some CPI calls that don't pass that data (ironically upload_stemcell is one of them). But technically the `['properties']` are still a "more correct" looking set of config params.